packages/app-project/src/screens/ClassifyPage/components/YourProjectStats/helpers/dateRangeHelpers.js

```diff
@@ -0,0 +1,21 @@
+/* Today in UTC */
+export function getTodayDateString() {
```

Comment:
For the helper functions in this file, I considered importing similar functions from lib-user, but the hooks and helpers in lib-user are specific to components that handle stats counts and hours and multiple date ranges. This app-project YourProjectStats doesn't need all the bells and whistles like lib-user, so I decided to define simplified helper functions here.